### PR TITLE
Add postman collection and do do a general cleanup

### PR DESCRIPTION
**Pull request by SMLuthi**
### Fixes [DPC-631](https://jiraent.cms.gov/browse/DPC-631)

We want to move the Richards postman collection to our own repository so we can continue to maintain it.

### Proposed Changes

Move the postman collection over and do a general clean up.

### Change Details

The postman collection has been moved and the following has been applied:

- Removed the `Usage: Configuration`, `Not Supported Yet`, and `SMART Backend Services: DPC API Access Token from JWT` sections
- Removed all mention of Richards name
- Replaced hardcoded key id in `Remove/Delete Public Key from Server` and replaced with env var `key_id`
- Set expiration date to a generic default value

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by Sun-Mountain — APPROVED** [review]
👍

File: postman/Data at the Point of Care Sandbox.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			"enabled": true
+		},
+		{
+			"key": "provider_npi",
+			"value": "1234329724",
+			"enabled": true
+		},
+		{
+			"key": "group-id",
+			"value": "9aed444d-e9e4-4dfe-a7fa-044189b88116",
```
Comment (dhgreene) [inline postman/Data at the Point of Care Sandbox.postman_environment.json:52]: . . . or this

File: postman/Data at the Point of Care Sandbox.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+			"enabled": true
+		},
+		{
+			"key": "attestationHeader",
+			"value": "{\"resourceType\":\"Provenance\",\"recorded\":\"2020-06-03T16:44:45.008Z\",\"reason\":[{\"system\":\"http://hl7.org/fhir/v3/ActReason\",\"code\":\"TREAT\"}],\"agent\":[{\"role\":[{\"coding\":[{\"system\":\"http://hl7.org/fhir/v3/RoleClass\",\"code\":\"AGNT\"}]}],\"whoReference\":{\"reference\":\"Organization/351fbb5f-f2f9-4094-bc6f-2b3600bb56e9\"},\"onBehalfOfReference\":{\"reference\":\"Practitioner/0c527d2e-2e8a-4808-b11d-0fa06baf8254\"}}]}",
+			"enabled": true
+		},
+		{
+			"key": "practitioner-id",
+			"value": "0c527d2e-2e8a-4808-b11d-0fa06baf8254",
```
Comment (dhgreene) [inline postman/Data at the Point of Care Sandbox.postman_environment.json:42]: I don't think having an example value is useful for anyone

File: postman/Data at the Point of Care Sandbox.postman_environment.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+			"enabled": true
+		},
+		{
+			"key": "group-id",
+			"value": "9aed444d-e9e4-4dfe-a7fa-044189b88116",
+			"enabled": true
+		},
+		{
+			"key": "patient-id",
+			"value": "995a1c0f-b6bc-4d16-b6b0-b8a6597c6e1d",
```
Comment (dhgreene) [inline postman/Data at the Point of Care Sandbox.postman_environment.json:57]: . . . or this

File: postman/Data at the Point of Care Sandbox.postman_environment.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+			"enabled": true
+		},
+		{
+			"key": "patient_mbi",
+			"value": "4S41C00AA00",
+			"enabled": true
+		},
+		{
+			"key": "organization-id",
+			"value": "351fbb5f-f2f9-4094-bc6f-2b3600bb56e9",
```
Comment (dhgreene) [inline postman/Data at the Point of Care Sandbox.postman_environment.json:77]: . . . or this

**Review by dhgreene — COMMENTED** [review]
Looking good!  Just a few deletions (though keep the keys with blank values).

**Pull request merged**